releases/release-1.25/release-notes/maps/pr-109251-map.yaml

```diff
@@ -1,5 +1,6 @@
 pr: 109251
-releasenote:
-  text: `MaxUnavailable` for `StatefulSets`, allows faster `RollingUpdate` by taking down
-    more than 1 pod at a time. The number of pods you want to take down during a `RollingUpdate`
-    is configurable using `maxUnavailable` parameter.
```

Comment:
I think this has been generated from the krel input, right @csantanapr? If so, then we may have a bug here.

Reply:
@saschagrunert, this text was modified during PR review process by a person (ie me)

Reply:
I don't see what the error `decoding note map: yaml: line 3: found character that cannot start any token` is referring to